var/spack/repos/builtin/packages/py-yt/package.py

```diff
@@ -51,8 +53,8 @@ class PyYt(PythonPackage):
             tag="yt-3.0.2", commit="511887af4c995a78fe606e58ce8162c88380ecdc")
     version("2.6.3", hg="https://bitbucket.org/yt_analysis/yt",
             tag="yt-2.6.3", commit="816186f16396a16853810ac9ebcde5057d8d5b1a")
-    version("development", hg="https://bitbucket.org/yt_analysis/yt",
-            branch="yt")
+    version("development", git="https://github.com/yt-project/yt.git",
```

Comment:
Should it be `develop` to be consistent with other packages?

Reply:
Yes that should be better.